src/core/manager.c

```diff
@@ -3904,18 +3904,17 @@ static void manager_send_ready_user_scope(Manager *m) {
         m->status_ready = false;
 }
 
-static void manager_send_ready_system_scope(Manager *m) {
+static void manager_send_ready_finished(Manager *m) {
```

Comment:
hmm, let's maybe rename both functions then after what they do, i.e. rename

1. manager_send_ready_user_scope() → manager_send_ready_on_basic_targe()
2. manager_send_ready_system_scope() → manager_send_ready_on_idle()

I think this would make things a lot clearer at what happens.